api/tests/test_hyp3_api.py

```diff
@@ -23,48 +24,23 @@ def client():
 
 @pytest.fixture
 def table():
+    table_properties = get_table_properties_from_template()
     with mock_dynamodb2():
         table = DYNAMODB_RESOURCE.create_table(
             TableName=environ['TABLE_NAME'],
-            AttributeDefinitions=[
-                {
-                    'AttributeName': 'job_id',
-                    'AttributeType': 'S'
-                },
-                {
-                    'AttributeName': 'user_id',
-                    'AttributeType': 'S',
-                },
-
-            ],
-            KeySchema=[
-                {
-                    'AttributeName': 'job_id',
-                    'KeyType': 'HASH',
-                },
-            ],
-            GlobalSecondaryIndexes=[
-                {
-                    'IndexName': 'user_id',
-                    'KeySchema': [
-                        {
-                            'AttributeName': 'user_id',
-                            'KeyType': 'HASH',
-                        },
-                    ],
-                    'Projection': {
-                        'ProjectionType': 'ALL',
-                    },
-                },
-            ],
-            ProvisionedThroughput={
-                'ReadCapacityUnits': 15,
-                'WriteCapacityUnits': 15,
-            },
+            **table_properties,
         )
         yield table
 
 
+def get_table_properties_from_template():
+    yaml.SafeLoader.add_multi_constructor('!', lambda loader, suffix, node: None)
+    with open('../cloudformation.yml', 'r') as f:
```

Comment:
This is from the working directory, which with pytest could be anywhere.  If we want to pursue this approach we'll need to make it relative to the `test_hyp3_api.py` file's directory, or load it in explicitly as an environment variable.

Reply:
I think os.path has a tool for this?